### ISSUE TITLE
NVDA cannot read the error message from validateInput in vscode.window.showInputBox

### ISSUE THREAD

**Issue by IlyaBiryukov** [microsoft/vscode#55186]
Issue Type: <b>Bug</b>

An extension calls `vscode.window.showInputBox` and provides `validateInput` option. The user enters some value which triggers `validateInput` and make it return an error message. The user tries to use NVDA to read the error message.

Expected: NVDA can read the error message.

Actual: NVDA is not reading the error message.

May be a dupe of #41406

VS Code version: Code 1.25.0 (0f080e5267e829de46638128001aeb7ca2d6d50e, 2018-07-05T13:11:58.697Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz (8 x 3408)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.92GB (24.06GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
cpptools|ms-|0.17.7
vsliveshare|ms-|0.3.472

</details>

**Comment by isidorn** [comment]
@chrmarti I think this might be fixed now?

**Comment by chrmarti** [comment]
Indeed, this is fixed. Thanks.
